tox.ini

```diff
@@ -2,6 +2,7 @@
 minversion = 3.9.0
 envlist = py39, py310, py311, flake8, mypy
 isolated_build = true
+toxworkdir = {toxinidir}/../.tox
```

Comment:
By default, tox places all of its temporary test files within the project dir. This lead to an issue where all of the 'temp tests' generated by my plugin tests were invoking the project's pyproject.toml file and deactivating the retry plugin. Setting the .tox dir somewhere outside the project dir removes the pyproject.toml from the temp test file hierarchy and solves the issue.